domain/transfer/sender/service.go

```diff
@@ -718,6 +709,37 @@ func (s service) AssignTransfer(ctx context.Context, customerID int, transferID,
 	return negotiation, err
 }
 
+// createAuthCredentials creates 2 authorization credentials, one for the Task, and one for the nursingHandoffComposition.
+func (s service) createAuthCredentials(ctx context.Context, transferTask *eoverdracht.TransferTask, nursingHandoffComposition *fhir.Composition, customerDID, organizationDID string) error {
+	// Create an Auth Credential for the Task
+	authorizedTask := s.taskForNursingHandoff(transferTask.ID)
+	if err := s.vcr.CreateAuthorizationCredential(ctx, customerDID, &credential.NutsAuthorizationCredentialSubject{
+		ID: organizationDID,
+		LegalBase: credential.LegalBase{
+			ConsentType: "implied",
+		},
+		PurposeOfUse: transfer.SenderServiceName,
+		Resources:    authorizedTask,
+	}); err != nil {
```

Comment:
The `if` initializer is not well suited for multiline constructions. 😄 Kudos for how easy this methods reads though.